src/mvba/consensus.rs

```diff
@@ -72,8 +71,8 @@ impl Consensus {
         }
     }
 
-    // start the consensus by proposing a proposal and broadcasting it.
-    pub fn start(&mut self, _proposal: Proposal) -> Vec<Vec<u8>> {
+    /// starts the consensus by broadcasting the `m`.
+    pub fn start(&mut self, _m: &[u8]) -> Vec<Vec<u8>> {
```

Comment:
let's use the proposal alias here since we have it defined.
```suggestion
    pub fn start(&mut self, _proposal: Proposal) -> Vec<Vec<u8>> {
```